src/test/java/org/apache/datasketches/kll/KllFloatsSketchSerDeTest.java

```diff
@@ -64,7 +64,7 @@ public void serializeDeserializeOneValue() {
     assertEquals(sketch2.getNormalizedRankError(false), sketch1.getNormalizedRankError(false));
     assertEquals(sketch2.getMinItem(), 1.0F);
     assertEquals(sketch2.getMaxItem(), 1.0F);
-    assertEquals(sketch2.getSerializedSizeBytes(), 8 + Float.BYTES);
+    assertEquals(sketch2.getSerializedSizeBytes(), sketch2.getSerializedSizeBytes());
```

Comment:
this is always true. perhaps something else was meant?

Reply:
Oops!  I should have just left this one alone.  I was trying to eliminate the magic number :(